CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 Changelog
 =========
 
-[v2.15](https://github.com/rigetti/pyquil/compare/v2.14.0...master) (in development)
-------------------------------------------------------------------------------------
+[v2.15](https://github.com/rigetti/pyquil/compare/v2.14.0...master) (December 20, 2019)
```

Comment:
should "master" in the revision range change to v2.15.0 or does that happen after the release?

Reply:
oh I just saw the wip label nevermind

Reply:
I normally do it after the release because right now there is no `v2.15.0` tag